src/lib/viewers/controls/fullscreen/FullscreenToggle.tsx

```diff
@@ -5,7 +5,7 @@ import useFullscreen from '../hooks/useFullscreen';
 import './FullscreenToggle.scss';
 
 export type Props = {
-    onFullscreenToggle: (isFullscreen: boolean) => void;
+    onFullscreenToggle: (isFullscreen: boolean, toggleFullscreen: EventTarget | null) => void;
```

Comment:
should it be `toggleFullscreenEl` to be consistent?

Reply:
Was there a commit to address this comment?

Reply:
Sorry, updated!